timemachine/cpp/src/kernels/k_nonbonded.cuh

```diff
@@ -54,6 +54,38 @@ void __global__ k_check_rebuild_coords_and_box_gather(
     }
 }
 
+template <typename RealType, int COORDS_DIM, int PARAMS_DIM>
+void __global__ k_rescale_coords(
+    const int NUM_ROW_ATOMS,
+    const unsigned int *__restrict__ row_atom_idxs,
+    const unsigned int *__restrict__ row_anchor_idxs,
```

Comment:
nit: could this just be `NUM_ATOMS`, `atom_idxs`, `anchor_idxs`? (seems orthogonal to concept of "row" atoms)

Reply:
(stale)